src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderAssessmentResource.java

```diff
@@ -42,18 +42,19 @@
             @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class, responseContainer = "List")})
     List<Assessment> getOffenderAssessmentsAssessmentCode(@ApiParam(value = "Assessment Type Code", required = true) @PathVariable("assessmentCode") String assessmentCode,
                                                           @ApiParam(value = "The required offender numbers", required = true) @RequestParam("offenderNo") @NotEmpty @RequestBody List<String> offenderNo,
-                                                          @ApiParam(value = "Returns only the latest assessment for the current sentence if true, otherwise all assessments for all previous sentences are included", defaultValue = "true") @RequestParam(value = "latestOnly", required = false, defaultValue = "true") Boolean latestOnly,
-                                                          @ApiParam(value = "Returns only active assessments if true, otherwise assessments with any status are included", defaultValue = "true") @RequestParam(value = "activeOnly", required = false, defaultValue = "true") Boolean activeOnly);
+                                                          @ApiParam(value = "Returns only assessments for the current sentence if true, otherwise assessments for all previous sentences are included", defaultValue = "true") @RequestParam(value = "latestOnly", required = false, defaultValue = "true") Boolean latestOnly,
+                                                          @ApiParam(value = "Returns only active assessments if true, otherwise inactive and pending assessments are included", defaultValue = "true") @RequestParam(value = "activeOnly", required = false, defaultValue = "true") Boolean activeOnly,
```

Comment:
Have you checked for other clients of this API to make sure they didn't rely on the old functionality?  (and for other endpoints changed)